### ISSUE TITLE
Could not generate images from .mmd files including non-ascii characters.

### ISSUE THREAD

**Issue by ndac-todoroki** [tomoyukim/vscode-mermaid-editor#27]
An example mermaid file:
```mmd
sequenceDiagram
    participant x as Aさん
    participant y as Bさん

    x ->> y: プレゼントを渡す
```

The preview works: 
![image](https://user-images.githubusercontent.com/11530165/101805280-eaa6f000-3b55-11eb-81c5-24aa0c969b37.png)

But not the generator:

```
Parse error on line 1:
#mermaid-16076197498
^
Expecting 'NEWLINE', 'SPACE', 'GRAPH', got 'BRKT'

============================
```

Changing all Japanese words to ascii letters makes the generator work properly.

**Comment by tomoyukim** [comment]
Thank you for reporting the bug. This seems to be caused by same reason as #29.

**Comment by tomoyukim** [comment]
Thank you for waiting. This should be resolved in v0.9.0.
